### PR TITLE
Add configuration option for coriolis.io instance to use

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**

### CODE REVIEW

**Comment by A-UNDERSCORE-D** [comment]
Added a dropdown to select which one to go with, if not auto.

File: plugins/coriolis.py [hunk 1]
```diff
@@ -13,13 +22,101 @@
 
 config.delete('shipyard', suppress=True)
 
+logger = get_main_logger()
+
+DEFAULT_NORMAL_URL = 'https://coriolis.io/import?data='
+DEFAULT_BETA_URL = 'https://beta.coriolis.io/import?data='
+DEFAULT_OVERRIDE_MODE = 'auto'
```
Comment (Athanasius) [inline plugins/coriolis.py:29]: This starts out as 'auto' in the dropdown, but in the dropdown there's "normal", "beta" and **"default"**.  So one of 'auto' or 'default' needs to change.

File: plugins/coriolis.py [hunk 2]
```diff
@@ -13,13 +22,101 @@
...
+    cur_row += 1
+
+    nb.Label(conf_frame, text=_('Normal URL')).grid(sticky=tk.W, row=cur_row, column=0, padx=PADX)
+    nb.Entry(conf_frame, textvariable=normal_textvar).grid(sticky=tk.EW, row=cur_row, column=1, padx=PADX)
+    cur_row += 1
+
+    nb.Label(conf_frame, text=_('Beta URL')).grid(sticky=tk.W, row=cur_row, column=0, padx=PADX)
+    nb.Entry(conf_frame, textvariable=beta_textvar).grid(sticky=tk.EW, row=cur_row, column=1, padx=PADX)
+    cur_row += 1
+
```
Comment (Athanasius) [inline plugins/coriolis.py:73]: These each need a 'Default' button on the right hand side to make it *easy* to revert.

File: plugins/coriolis.py (hunk 1 above)
Comment (A-UNDERSCORE-D) [reply plugins/coriolis.py:29]: whoops, it should be auto

File: plugins/coriolis.py (hunk 2 above)
Comment (A-UNDERSCORE-D) [reply plugins/coriolis.py:80]: Added

File: plugins/coriolis.py (hunk 1 above)
Comment (A-UNDERSCORE-D) [reply plugins/coriolis.py:30]: Changed

Comment (Athanasius) [reply plugins/coriolis.py:30]: s/auto/Auto/ - else first time you look it shows this "auto" as the current value, when it's "Auto" in the dropdown.

File: plugins/coriolis.py [hunk 3]
```diff
@@ -13,13 +23,120 @@
...
+def _get_target_url(is_beta: bool) -> str:
+    global override_mode
+    if override_mode not in ('auto', 'normal', 'beta'):
+        show_error(_('Invalid Coriolis override mode!'))
+        logger.warning(f'Unexpected override mode {override_mode!r}! defaulting to auto!')
+        override_mode = 'auto'
+
+    if override_mode == 'beta':
+        return beta_url
+    elif override_mode == 'normal':
```
Comment (Athanasius) [inline plugins/coriolis.py:125]: \*gasp\* - no blank line ?

File: L10n/en.template [hunk 4]
```diff
@@ -601,3 +601,30 @@
...
+"Reset" = "Reset";
+
+/* Coriolis Normal URL label [plugins/coriolis.py] */
+"Normal URL" = "Normal URL";
+
+/* Coriolis Beta URL label [plugins/coriolis.py] */
+"Beta URL" = "Beta URL";
+
+/* Coriolis config title label [plugins/coriolis.py] */
+"Set the URL to use with coriolis.io ship loadouts. Note that this MUST end with 'import?data='" = "Set the URL to use with coriolis.io ship loadouts. Note that this MUST end with 'import?data='";
```
Comment (Athanasius) [inline L10n/en.template:627]: The code has `/import` here, with the `/`.

File: plugins/coriolis.py [hunk 5]
```diff
@@ -13,13 +23,121 @@
...
+    conf_frame.columnconfigure(index=1, weight=1)
+    cur_row = 0
+    nb.Label(conf_frame, text=_(
+        "Set the URL to use with coriolis.io ship loadouts. Note that this MUST end with '/import?data='"
+    )).grid(sticky=tk.EW, row=cur_row, column=0, columnspan=3)
+    cur_row += 1
+
+    nb.Label(conf_frame, text=_('Normal URL')).grid(sticky=tk.W, row=cur_row, column=0, padx=PADX)
+    nb.Entry(conf_frame, textvariable=normal_textvar).grid(sticky=tk.EW, row=cur_row, column=1, padx=PADX)
+    nb.Button(conf_frame, text=_("Reset"), command=lambda: normal_textvar.set(value=DEFAULT_NORMAL_URL)).grid(
```
Comment (Athanasius) [inline plugins/coriolis.py:71]: `Reset` needs adding to translation template.

Comment (A-UNDERSCORE-D) [reply plugins/coriolis.py:71]: its in there

**Pull request merged**